solr/README.md

```diff
@@ -117,6 +117,43 @@ List of filters is only supported in Datadog Agent > 5.3.0. If you are using an
 
 For containerized environments, see the [Autodiscovery with JMX][2] guide.
 
+##### Log collection
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+      ```yaml
+       logs_enabled: true
+     ```
+
+2. Solr uses the `log4j` logger by default. To customize the logging format, edit the [`server/resources/log4j2.xml`][12] file. By default, Datadog's integration pipeline supports the following conversion [pattern][11]:
+
+   ```text
+   %maxLen{%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p (%t) [%X{collection} %X{shard} %X{replica} %X{core}] %c{1.} %m%notEmpty{ =>%ex{short}}}{10240}%n
+   ```
+
+    Clone and edit the [integration pipeline][13] if you have a different format.
+
+
+3. Uncomment and edit the logs configuration block in your `solr.d/conf.yaml` file. Change the `type`, `path`, and `service` parameter values based on your environment. See the [sample solr.d/solr.yaml][6] for all available configuration options.
+
+      ```yaml
+       logs:
+         - type: file
+           path: /var/solr/logs/solr.log
+           source: solr
+           service: <SERVICE_NAME>
```

Comment:
```suggestion
```
The recommended way now is to use the service config option

Reply:
Updated!